octokit/search.go

```diff
@@ -5,44 +5,44 @@ import (
 	"net/url"
 )
 
-var SearchURL = Hyperlink("search{/type}?q={query}{&page,per_page,sort,order}")
+var SearchURL = Hyperlink("search{/type}?q={query}&page={page}&per_page={per_page}&sort={sort}&order={order}")
```

Comment:
No need to change these. That's the correct format for URI template params like this. The expansion will handle the `?` and `&` automatically.